src/main/java/de/ait/secondlife/services/offer_status/AuctionFinishedState.java

```diff
@@ -0,0 +1,91 @@
+package de.ait.secondlife.services.offer_status;
+
+import de.ait.secondlife.domain.constants.OfferStatus;
+import de.ait.secondlife.domain.entity.Bid;
+import de.ait.secondlife.domain.entity.Offer;
+import de.ait.secondlife.exception_handling.exceptions.ProhibitedOfferStateChangeException;
+import de.ait.secondlife.services.interfaces.OfferService;
+
+import java.util.List;
+
+public class AuctionFinishedState extends StateStrategy {
```

Comment:
Why the class AuctionFinishedState contains methods regarding previous states? Need to talk about it. Dont understand